man/journalctl.xml

```diff
@@ -854,6 +854,17 @@
         cryptographic theory it is based on.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--convert=</option></term>
+
+        <listitem><para>Converts the specified journal files to the latest supported journal format. Takes
+        the path to store the converted journal files. The path should include the filename to be used for
+        the converted files, with the <literal>.journal</literal> extension (e.g.
+        <filename>/a/b/c/converted.journal</filename> will store the journal files in the
+        <filename>/a/b/c</filename> directory using <filename>converted.journal</filename> as the filename).
+        </para></listitem>
```

Comment:
I find this very confusing? "The specified journal files"? what does that mean, which ones? If I invoke "journalctl --convert=foo.journal" then I am not specifiying any source files.

What about calling this "journalctl --rewrite=…" or so? because you are not really just converting, you are entirely rewriting stuff, i.e. journal files might be differently split up afterwards.

And maybe say the path specified is the output journal file path, but if rotation is taking place files will be placed to the specified file in the same directory.

Reply:
I removed this commit from the PR, I'll prepare a separate PR for it with the requested fixed